Source/Fuse.Elements/Resources/HttpImageSource.uno

```diff
@@ -143,7 +152,13 @@ namespace Fuse.Resources
 			else
 				_contentType = ct;
 
-			new BackgroundLoad(data, _contentType, SuccessCallback, FailureCallback);
+			try
+			{
+			_orientation = ExifData.FromByteArray(data).Orientation;
+			}
+			//If Exif is not supported we just stick with the default (ImageOrientation.Identity)
+			catch (Exception e) { }
```

Comment:
The indentation here seems wonky. And we'll leak the orientation of the previously loaded image here, if the image got loaded, switched on the server to a differently oriented version, and then reloaded (there's a few ways this can happen). So as-is, this isn't robust. We need to assign _orientation in all cases.

Also, since this is needed in multiple places, how about making a helper for it?

Something like:
```uno
ImageOrientation GetImageOrientationFromByteArrayOrDefault()
{
	try
	{
		return ExifData.FromByteArray(data).Orientation;
	}
	catch (Exception e)
	{
		return ImageOrientation.Identity;
	}
}
```

But thinking about it a bit more, it feels pretty wasteful to even raise an exception if all we're going to do is ignore the exception. So perhaps we should have a non-throwing API here instead? Something like `static bool TryFromByteArray(byte[] data, out ExifData exifData);`? That would make the helper 

```uno
ImageOrientation GetImageOrientationFromByteArrayOrDefault()
{
	ImageOrientation orientation;
	if (!ExifData.TryFromByteArray(data, out orientation))
		return ImageOrientation.Identity;

	return orientation;
}
```

Reply:
Hmm, i realize that ImageOrientation could just default to Identity, so do we really need the bool? I guess it will be more relevant if we add support for more exif tags.